### DISCUSSION TITLE
Upload file and its dependent files thru FTP/SFTP

### DISCUSSION THREAD

**Discussion by chk1xn** [microsoft/vscode-discussions#877] [Extension Development QnA] [answered]
I'm new to VSC and have currently SFTP by Natizyskun added to my installation. Is there an option in this or other extensions that would not only upload a modified file but also checks if all files it is referencing to (css, js, images, etc,) have changed (or haven't been uploaded yet at all) and so need to be uploaded too?

(Coming from Dreamweaver I found this to be a very handy option to select when uploading files to a SSH server...)

Thanks.

**Comment by gjsjohnmurray** [comment, marked as answer]
Please ask the extension developer directly. And questions about whether other suitable extensions exist are better posted on Stackoverflow. This forum is currently intended for peer to peer support amongst people who are building extensions.

**Reply by chk1xn** [reply]
Thank you. So much to learn, I'll try my best. :-)
